packages/docs/content/components/form-label.mdx

```diff
@@ -34,37 +34,49 @@ The following Sass variables can be overridden to customize Form components:
 
 ## Guidance
 
-### When to use
-
-#### Labels
-
-- Each form field should have a `<label>`. Never use a field's placeholder attribute as the primary way to label the field.
-
-#### Legends
+### Form legend
 
 - Use a single legend for fieldset (this is required). One example of a common use of fieldset and legend is a question with radio button options for answers. The question text and radio buttons are wrapped in a fieldset, with the question itself being inside the legend tag.
+- Avoid links in legends when possible. Links inside legends are not read by some screen readers, most notably Internet Explorer 11, paired with JAWS. If you need a pattern like this, put the link outside of the legend element, but keep all other useful hint text in the legend.
```

Comment:
Is this valid now that we no longer support IE 11? @SparrowSpit, do you know of any other software combos that would make this necessary? Or is it possibly a bad pattern in general?

Reply:
[It looks like this is a known issue](https://github.com/FreedomScientific/VFO-standards-support/issues/402) and I also tested and experienced the issue described in that convo. 

> You tab into a button within the element. The legend button is read.
You tab into a button underneath the contained within the fieldset. The fieldset button is read.
You shift-tab back to the button within the element.
The previous button is read back to you.

Reply:
Okay, so it's also a problem for Chrome, Firefox, and Edge according to that ticket. Thanks, @SparrowSpit! Maybe we should update it to say 
```suggestion
- Avoid links in legends when possible. Links inside legends are not read by some screen readers, namely JAWS. If you need a pattern like this, put the link outside of the legend element, but keep all other useful hint text in the legend.
```